src/NServiceBus.Core/Hosting/StartupDiagnostics/EndpointDiagnosticSettingsExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace NServiceBus
+{
+    using Settings;
+
+    /// <summary>
+    /// Provides an API to add startup diagnostics.
+    /// </summary>
+    static class EndpointDiagnosticSettingsExtensions
```

Comment:
Is this supposed to be public so downstreams can use it? If not, why have the xml comments?

Reply:
Good catch, fixed